src/core/observables/ParticleTraits.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2010-2020 The ESPResSo project
+ *
+ * This file is part of ESPResSo.
+ *
+ * ESPResSo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * ESPResSo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef OBSERVABLES_PARTICLE_TRAITS
+#define OBSERVABLES_PARTICLE_TRAITS
+
+#include "Particle.hpp"
+#include "config.hpp"
+
+namespace GenObs {
+template <> struct traits<Particle> {
```

Comment:
Some sort of documentation is needed here, i.e., the purpose of the traits class and what we are using it for